api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -69,6 +75,41 @@ public ApiClient allOfUsApiClient(WorkbenchConfig workbenchConfig) {
     return apiClient;
   }
 
+  @Bean(name = WGS_COHORT_EXTRACTION_SERVICE_ACCOUNT_API_CLIENT)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public ApiClient wgsExtractionServiceAccountApiClient(
+      WorkbenchConfig workbenchConfig, IamCredentialsClient iamCredentialsClient) {
+    ApiClient apiClient = buildApiClient(workbenchConfig);
+    apiClient.setAccessToken(
+        iamCredentialsClient
+            .generateAccessToken(
+                "projects/-/serviceAccounts/" + workbenchConfig.wgsCohortExtraction.serviceAccount,
+                Collections.emptyList(),
+                BILLING_SCOPES,
+                Duration.newBuilder().setSeconds(60).build())
```

Comment:
Hm. This could potentially cause issues if we ever decide to use this API client during a cron job (can run up to 10m). Do we have this issue elsewhere as well? Possibly could just increase